seeker.py

```diff
@@ -112,20 +126,152 @@ def banner():
 		json_data = loads(metadata.read())
 		twitter_url = json_data['twitter']
 		comms_url = json_data['comms']
-
-	art = r'''
-                        __
-  ______  ____   ____  |  | __  ____ _______
- /  ___/_/ __ \_/ __ \ |  |/ /_/ __ \\_  __ \
- \___ \ \  ___/\  ___/ |    < \  ___/ |  | \/
-/____  > \___  >\___  >|__|_ \ \___  >|__|
-     \/      \/     \/      \/     \/'''
+	arts = [
```

Comment:
this change is unnecessary, please revert this, multiple ascii art are not required so they wont be added